l5kit/l5kit/rasterization/semantic_rasterizer.py

```diff
@@ -130,31 +127,30 @@ def rasterize(
         # TODO TR_FACES
```

Comment:
Note the todo here! is it okay that this is still in? Also shouldn't it be TL_Faces?

Reply:
legacy, we can remove it